src/java/org/apache/cassandra/tools/nodetool/SetInterDCStreamThroughput.java

```diff
@@ -23,11 +23,11 @@
 import org.apache.cassandra.tools.NodeProbe;
 import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
 
-@Command(name = "setinterdcstreamthroughput", description = "Set the Mb/s throughput cap for inter-datacenter streaming and entire SSTable inter-datacenter streaming in the system, or 0 to disable throttling")
+@Command(name = "setinterdcstreamthroughput", description = "Set the megabits per second throughput cap for inter-datacenter streaming and entire SSTable inter-datacenter streaming in the system, or 0 to disable throttling")
```

Comment:
nit: It looks like it's `MiB/s` below?

Reply:
It should be megabits as this is a released property pre-4.0 Seems I changed it by mistake down.
Getting it back to` @Arguments(title = "inter_dc_stream_throughput", usage = "<value_in_mb>", description = "Value in megabits, 0 to disable throttling", required = true)`